src/main/java/org/rumbledb/cli/Main.java

```diff
@@ -74,24 +77,29 @@ public static void main(String[] args) throws IOException {
                 );
             }
         } catch (Exception ex) {
-            handleException(ex);
+            handleException(ex, sparksoniqConf.getErrorInfo());
         }
     }
 
-    private static void handleException(Throwable ex) {
+    private static void handleException(Throwable ex, boolean errorInfo) {
```

Comment:
As discussed in SparksoniqRuntimeConfiguration,  in my opinion, 'showErrorInfo' would serve as a much better boolean name. I would expect a method call such as handleException(...., errorInfo) to pass in the actual information, rather than the flag indicating whether info should be displayed or not.

Reply:
Also agreed